content/reference/api/hub/latest.yaml

```diff
@@ -6,112 +6,104 @@ info:
     url: https://docs.docker.com/assets/images/logo-docker-main.png
     href: /reference
   description: |
-    Docker Hub is a service provided by Docker for finding and sharing container
-    images with your team.
+    Docker Hub is a service provided by Docker for finding and sharing container images with your team.
 
     It is the world's largest library and community for container images.
 
-    In addition to the [Docker Hub UI](https://docs.docker.com/docker-hub/) and [Docker Hub CLI tool](https://github.com/docker/hub-tool#readme) (currently experimental),
-
-    Docker provides an API that allows you to interact with Docker Hub.
+    In addition to the [Docker Hub UI](https://docs.docker.com/docker-hub/) and [Docker Hub CLI tool](https://github.com/docker/hub-tool#readme) (currently experimental), Docker provides an API that allows you to interact with Docker Hub.
 
     Browse through the Docker Hub API documentation to explore the supported endpoints.
-
+servers:
+  - description: Docker HUB API
+    x-audience: public
+    url: https://hub.docker.com
 tags:
   - name: resources
     x-displayName: Resources
     description: |
       The following resources are available to interact with the documented API:
-
-      - <a href="https://github.com/docker/hub-tool#readme">Docker Hub CLI tool</a> (currently experimental)
+      - [Docker Hub CLI tool]("https://github.com/docker/hub-tool#readme) (currently experimental)
```

Comment:
this goes to a 404

Reply:
Doesn't for me 🤔

Reply:
weird. if I copy and paste the URL it is fine, but 404s when I click the link in the docs preview

Reply:
Hmm can dig a bit further here

Reply:
Fixed